InjectionIII/InjectionServer.mm

```diff
@@ -27,6 +27,7 @@
 @implementation InjectionServer {
     void (^injector)(NSArray *changed);
     FileWatcher *fileWatcher;
+    NSMutableArray *pending;
```

Comment:
Should we use a `set` so that we don't get multiple injections for the same file?

Reply:
You could but I de-dupe later on. I wanted to make sure the files came through in the original order. Would this be guaranteed with a set?

Reply:
Well not really, sets are unordered but they do remove duplicates.
Let's stick with this now and do another PR when we find an optimal solution.